.github/workflows/test.yml

```diff
@@ -209,7 +209,7 @@ jobs:
     timeout-minutes: 40
     strategy:
       matrix:
-        go-version: ["1.20.x", "1.21.x"]
+        go-version: ["1.21.x"]
```

Comment:
`slices.Contains` requires at least go version 1.21

If you still want to support 1.20, I can reimplement this functionality